packages/apps/docs/src/components/Layout/basestyles.css.ts

```diff
@@ -83,12 +87,12 @@ export const baseGridClass = style([
 
     ...responsiveStyle({
       md: {
-        gridTemplateColumns: `1% ${$$leftSideWidth} minmax(auto, calc(${$$pageWidth} - ${$$leftSideWidth} - ${$$asideMenuWidthMDDefault})) ${$$asideMenuWidthMDDefault} 1%`,
+        gridTemplateColumns: `1% ${$$leftSideWidth} minmax(auto, calc(${$$pageWidth} - ${$$leftSideWidth} - ${$$asideMenuWidthMDDefault})) 1%`,
```

Comment:
$$$ 🤑  not the clearest code, but I will assume that it works

Reply:
😅